src/api/EmployerFavouritesApi/Controllers/ApprenticeshipsController.cs

```diff
@@ -100,11 +106,40 @@ public async Task<IActionResult> Put(string employerAccountId, [FromBody]List<Fa
 
                 return NoContent();
             }
+            catch (InvalidUkprnException ex)
+            {       
+                return BadRequest(ex);              
+            }
+            catch (InvalidApprenticeshipIdException ex)
+            {
+                return BadRequest(ex);
```

Comment:
Even though it's an internal api we should probably return the message not the exception so that the internals of the app aren't leaked e.g. stack traces.
```csharp
            {       
                return BadRequest(ex.Message);              
            }
```